receiver/sqlqueryreceiver/testdata/integration/Dockerfile.mysql

```diff
@@ -0,0 +1,8 @@
+FROM mysql
```

Comment:
We should probably pin to a specific version of the [image](https://hub.docker.com/_/mysql)? Like `mysql:8.0.32` or perhaps only pin the minor version with `mysql:8.0`?

Reply:
good idea